### PR TITLE
Blood: fixes #343

### LINKED ISSUE #343: Blood: Cinematic/Movie + Custom content/Addons
when setting up Nblood with the movie folder + custom content (texture load) it duplicates the number of textures every time you die until Nblood crashes:
Frist time:
Loding 1915 textures
Die 1:
Loading 1915 * 2 = 3830 textures
die 2:
Loading 1915 * 3 = 5745 textures
......
....
Nblood Crash: nblood.exe has triggered a breakpoint.
image

### PR DESCRIPTION
**Pull request by MexMeRcUrY**
fixes #343

### CODE REVIEW

File: source/blood/src/blood.cpp [hunk 1]
```diff
@@ -455,8 +455,13 @@ void PreloadCache(void)
     gSysRes.PrecacheSounds();
     gSoundRes.PrecacheSounds();
     if (MusicRestartsOnLoadToggle)
+    {
         sndTryPlaySpecialMusic(MUS_LOADING);
+    }
+   
+    nPrecacheCount = 0;
```
Comment (nukeykt) [inline source/blood/src/blood.cpp:462]: Move this to the `PreloadTiles`

File: source/blood/src/blood.cpp [hunk 2]
```diff
@@ -455,8 +456,12 @@ void PreloadCache(void)
     gSysRes.PrecacheSounds();
     gSoundRes.PrecacheSounds();
     if (MusicRestartsOnLoadToggle)
+    {
```
Comment (nukeykt) [inline source/blood/src/blood.cpp:459]: Please revert these braces and newlines

Comment (MexMeRcUrY) [reply source/blood/src/blood.cpp:459]: sorry for all this. im still learning.

Comment (MexMeRcUrY) [reply source/blood/src/blood.cpp:459]: sorry again.. i think i got it right this time. i quit trying the history and revert function. and i just copy the file from your repository and did the changes again.

Comment (MexMeRcUrY) [reply source/blood/src/blood.cpp:459]: if you need me to delete this and start over the branch and pull request so all this commit trash is not kept in the history let me know

**Comment by CommonLoon102** [comment]
Yeah, git can be hard :D

**Pull request closed without merging**